dashboard/config/linux/upstream-apparmor-kasan.config

```diff
@@ -8043,6 +8056,7 @@ CONFIG_OVERLAY_FS_INDEX=y
 # CONFIG_OVERLAY_FS_NFS_EXPORT is not set
 # CONFIG_OVERLAY_FS_XINO_AUTO is not set
 # CONFIG_OVERLAY_FS_METACOPY is not set
+# CONFIG_OVERLAY_FS_DEBUG is not set
```

Comment:
Looks like something we should enable:
```
Say Y here to enable extra debugging checks in overlayfs. 
```

Reply:
For most filesystems, we disable `_FS_DEBUG` as its reports tend to be not kernel bugs, but rather complaints about invalid inputs.

Reply:
I don't think there is any system there. Kernel DEBUG configs are all chaotic and random. What does it for this subsystem?

Reply:
Looks like it'd be immediately caused by writing to the loop device while mounting (which syzbot likes to do).

Reply:
Overlayfs is not block-devive-based filesystem, there is not block device to write to. The superblock must come from somewhere else.